lib/policies/msRestUserAgentPolicy.ts

```diff
@@ -1,20 +1,27 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
-import { BaseFilter } from "./baseFilter";
-import { WebResource } from "../webResource";
+import * as os from "os";
```

Comment:
Any sense of what the footprint is for this in the browser? Consumers can specify `node: { os: "empty" }` in their webpack.config.js but if they forget to do it their bundle will be a bit fatter than strictly necessary.

Reply:
I don't know what the footprint is, but this RequestPolicy should only be added in the Node.js scenario. I've updated the createdDefaultRequestPolicyCreators() method to only add this RequestPolicy in the Node.js scenario.